packages/http-client-python/generator/test/unbranded/mock_api_tests/test_encode_duration.py

```diff
@@ -0,0 +1,60 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import datetime
+
+import pytest
+from encode.duration import DurationClient
+from encode.duration.property.models import (
+    Int32SecondsDurationProperty,
+    ISO8601DurationProperty,
+    FloatSecondsDurationProperty,
+    DefaultDurationProperty,
+    FloatSecondsDurationArrayProperty,
+)
+
+
+@pytest.fixture
+def client():
+    with DurationClient() as client:
+        yield client
+
+
+def test_query(client: DurationClient):
+    client.query.default(input=datetime.timedelta(days=40))
+    client.query.iso8601(input=datetime.timedelta(days=40))
+    client.query.int32_seconds(input=36)
+    client.query.int32_seconds_array(input=[36, 47])
+    client.query.float_seconds(input=35.625)
+    client.query.float64_seconds(input=35.625)
+
+
+def test_property(client: DurationClient):
+    result = client.property.default(DefaultDurationProperty(value=datetime.timedelta(days=40)))
+    assert result.value == datetime.timedelta(days=40)
+    result = client.property.default(DefaultDurationProperty(value="P40D"))
+    assert result.value == datetime.timedelta(days=40)
+    result = client.property.iso8601(ISO8601DurationProperty(value=datetime.timedelta(days=40)))
+    assert result.value == datetime.timedelta(days=40)
+    result = client.property.iso8601(ISO8601DurationProperty(value="P40D"))
+    assert result.value == datetime.timedelta(days=40)
+    result = client.property.int32_seconds(Int32SecondsDurationProperty(value=36))
+    assert result.value == 36
+    result = client.property.float_seconds(FloatSecondsDurationProperty(value=35.625))
+    assert abs(result.value - 35.625) < 0.0001
+    result = client.property.float64_seconds(FloatSecondsDurationProperty(value=35.625))
+    assert abs(result.value - 35.625) < 0.0001
```

Comment:
why are we splitting the existing generic tests into azure and unbranded?

Reply:
see https://github.com/microsoft/typespec/pull/5574#discussion_r1915860510